app/controllers/questions_search_controller.rb

```diff
@@ -37,11 +37,11 @@ def typeahead
                   .order('node.nid DESC')
                   .limit(25)
     questions.each do |match|
-      link = "<span class='pull-left'><i data-url='" + match.path(:question) + 
-             "' class='fa fa-question-circle'></i> " + match.title + "</span>"
+      link = "<i data-url='" + match.path(:question) + 
+             "' class='fa fa-question-circle'></i> " + match.title
```

Comment:
This fixes the problem of dropdown link in the questions search box which is not working currently.